examples/express-sample/app.js

```diff
@@ -58,7 +47,9 @@ app.use(express.static(path.join(__dirname, "public")));
 app.use("/", routes);
 app.use("/users", users);
 
-// Add the Raygun Express handler
+// Add the Raygun error Express handler
 app.use(raygunClient.expressHandler);
```

Comment:
I agree that it (express naming usage)  is too tightly coupled at the moment. But until we see popular requests for other frameworks, I'm happy with the current implementation until we implement support for other frameworks. I think the new approach looks good (moving to `raygun.breadcrumbs.express.ts`).

But does a standalone node app need breadcrumbs tracked manually?

Reply:
> But does a standalone node app need breadcrumbs tracked manually?

Yes, users would need to manually add breadcrumbs.

The `expressBreadcrumbsHandler` only adds a single breadcrumb for requests, so it isn't a big difference nevertheless.